src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -402,6 +418,17 @@ public virtual Task DispatchEventAsync(ulong eventHandlerId, EventFieldInfo? fie
             _isBatchInProgress = true;
 
             task = callback.InvokeAsync(eventArgs);
+            if (quiesce)
+            {
+                if (_ongoingQuiescenceTask == null)
+                {
+                    _ongoingQuiescenceTask = task;
```

Comment:
Doesn't this need to have similar error handling to the branch below? Otherwise it looks like `ErrorBoundary` wouldn't work in this case.

Reply:
We should have an E2E test showing the error handling behavior for form posts (with and without `ErrorBoundary`).

Reply:
Actually, this made me realize that the logic was wrong. We should just throw in this case and let the exception bubble if we are dispatching with quiesce, as the quiesce task will bubble the error.